### PR TITLE
Add 4.7 channel

### PR DESCRIPTION
**Pull request by warrenvw**
As suggested, add the 4.7 channel.

### CODE REVIEW

File: olm-catalog/serverless-operator/project.yaml [hunk 1]
```diff
@@ -9,7 +9,7 @@ olm:
   channels:
     default: '4.6'
     list:
-      - '4.6'
+      - '4.6,4.7'
```
Comment (markusthoemmes) [inline olm-catalog/serverless-operator/project.yaml:12]: This must actually be an enumerated list. This breaks the code figuring out which channel to use for the subscription, see https://github.com/openshift-knative/serverless-operator/blob/28c6716b70c438a15ae747fd6c2e3ce7ed472350/hack/lib/vars.bash#L57.

File: olm-catalog/serverless-operator/project.yaml [hunk 2]
```diff
@@ -9,7 +9,8 @@ olm:
   channels:
     default: '4.6'
     list:
-      - '4.6'
+      - 4.6
+      - 4.7
```
Comment (cardil) [inline olm-catalog/serverless-operator/project.yaml:13]:
```suggestion
      - '4.6'
      - '4.7'
```
Without quotes they will be read as floats, but they should be strings. I advised adding quotes.

**Review by cardil — CHANGES_REQUESTED** [review]
Nit.

**Review by markusthoemmes — APPROVED** [review]
/lgtm
/approve

**Pull request merged**